ee/dtr/admin/manage-webhooks/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Manage webhooks
+description: Learn how to create, configure, and test webhooks in Docker Trusted Registry.
+keywords: registry, webhooks
+redirect_from:
+  - /datacenter/dtr/2.5/guides/user/create-and-manage-webhooks/
+  - /ee/dtr/user/create-and-manage-webhooks/
+---
+
+You can configure DTR to automatically post event notifications to a webhook URL of your choosing. This lets you build complex CI and CD pipelines with your Docker images. The following is a complete list of event types you can trigger webhook notifications for via the [web interface](use-the-web-ui) or the [API](use-the-API).
+
+## Event types for webhooks
+
+| Event Type                              | Scope                   | Access Level     | Availability |
+| --------------------------------------- | ----------------------- | ---------------- | ------------ |
+| Tag pushed to repository (`TAG_PUSH`)               | Individual repositories | Repository admin | Both        |
+| Tag pulled from repository (`TAG_PULL`)           | Individual repositories | Repository admin | Both        |
+| Tag deleted from repository (`TAG_DELETE`)            | Individual repositories | Repository admin | Both         |
+| Manifest pushed to repository (`MANIFEST_PUSH`)        | Individual repositories | Repository admin | Both         |
+| Manifest pulled from repository (`MANIFEST_PULL`)     | Individual repositories | Repository admin | Both         |
+| Manifest deleted from repository (`MANIFEST_DELETE`)      | Individual repositories | Repository admin | Both         |
+| Security scan completed (`SCAN_COMPLETED`)           | Individual repositories | Repository admin | Both         |
+| Security scan failed (`SCAN_FAILED`)                    | Individual repositories | Repository admin | Both         |
+| Image promoted from repository (`PROMOTION`)          | Individual repositories | Repository admin | Both         |
+| Image mirrored from repository (`PUSH_MIRRORING`)         | Individual repositories | Repository admin | Both         |
+| Image mirrored from remote repository (`POLL_MIRRORING`)   | Individual repositories | Repository admin | Both         |
+| Repository created, updated, or deleted (`REPO_EVENT`) | Namespace               | Namespace admin  | API Only     |
```

Comment:
@RobbKistler can you please review this table in particular? It's unclear to me what we're calling namespace/org admin based on my testing. If you create a new user on the UI without toggling "Is a trusted registry admin", then you do not get access to the Organizations view at all (unless you've been added to one). Also, we have different ways of referring to admins:
- global
- system
- security
- DTR / trusted registry (on user creation via the UI)

How are they distinguished exactly from an RBAC standpoint? Or are those different ways of labeling what we're calling "Trusted Registry"or "System" admins in the UI? I have not played around with user creation via the API. @jose-bigio, this was what I mentioned in our call earlier.

Reply:
Adding a user to an org does give you the "org owner" option. Is it safe to assume that's what we mean by "namespace" admins?